docs/how-to/watch-cou-logs.rst

```diff
@@ -0,0 +1,48 @@
+==============
+Watch COU logs
+==============
+
+Below is the script to watch the COU logs when doing the development.
+This script watches every log file in the directory, even newly created files.
+
+.. code:: bash
+
+    #!/bin/bash
+
+    # Change LOG_DIR to the target directory
+    DIRECTORY="~/.local/share/cou/log/"
```

Comment:
I tried this, and it didn't work with the tilde home shortcut unfortunately.

```suggestion
    DIRECTORY="$HOME/.local/share/cou/log/"
```

Reply:
Updated